package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@curriculumassociates/createjs-accessibility",
-  "version": "0.2.7",
+  "version": "1.0.0-prerelease.1",
```

Comment:
These are API breaking changes due to the splitting of the option role, so it requires a major version bump.  The pre-release is because there are other changes needed to CAM for it to graduate to a 1.0 version (e.g. finish implementing all the ARIA roles).